JsonLd/Serializer/DateTimeNormalizer.php

```diff
@@ -50,9 +54,30 @@ public function normalize($object, $format = null, array $context = [])
 
     /**
      * {@inheritdoc}
+     *
+     * @throws InvalidArgumentException Invalid date timezone or date format
```

Comment:
Hum, I don't like it at all. But that's basically Serializer interfaces' fault :)